driver/src/main/scala/org/mongodb/scala/model/package.scala

```diff
@@ -225,6 +243,28 @@ package object model {
   }
 
   /**
+   * The options to apply to an operation that inserts a single document into a collection.
+   *
+   * @Note Requires MongoDB 3.2 or greater
+   * @since 1.1
+   */
+  type InsertOneOptions = com.mongodb.client.model.InsertOneOptions
+
+  /**
+   * The options to apply to an operation that inserts a single document into a collection.
+   *
+   * @Note Requires MongoDB 3.2 or greater
+   * @since 1.1
+   */
+  object InsertOneOptions {
```

Comment:
Seems like this is included in the wrong commit: the one with the changes to Aggregates and Accumulators.